controllers/openshift_controller.go

```diff
@@ -771,6 +744,14 @@ func (r *KataConfigOpenShiftReconciler) processKataConfigDeleteRequest() (ctrl.R
 			}
 		}
 	}
+	err = r.unlabelNodes(&metav1.LabelSelector{MatchLabels: r.getNodeSelectorAsMapNoKataOc()})
+	if err != nil {
+		if k8serrors.IsConflict(err) {
+			return ctrl.Result{Requeue: true, RequeueAfter: 10 * time.Second}, nil
```

Comment:
nit: How about having the requeueAfter timeout as a constant for consistent usage across the code ?

Reply:
This will be dealt with explicitly and systematically across the controller code as part of KATA-1968.  At the moment though, I'd probably be in favour of using timeouts based on a particular use-case.

E.g. when requeueing to wait for the MCO 10-15 seconds might be reasonable (the MCO usually takes about 5 s to react).

In this particular case, I'd be actually inclined to use a much shorter wait, like a second or two since this is caused by the often-occurring "the object has been modified" error where long wait times don't seem to have much benefit (in fact, retrying right away, without deferring to the requeueing mechanism, would probably succeed AFAICT).

We should properly discuss this before work on KATA-1968 starts. I do agree that 10 s is kind of arbitrary but it's going to change.